### ISSUE TITLE
The loss of the validation set does not decrease

### ISSUE THREAD

**Issue by Wangweilai1** [huoyijie/AdvancedEAST#45]
Excuse me , Use you ICPR dataset and run python preprocess.py && python label.py and python advanced_east.py, but the validation loss haven't decrease, so when sotp, Test data is very bad.
can you help me,What went wrong?

**Comment by Taylor-Rose** [comment]
I think when the loss decreased to 0.02 or lower then the model can be used, but it's seems it's difficult, have you solved it?
